internal/dinosql/parser.go

```diff
@@ -382,7 +382,7 @@ func parseQuery(c core.Catalog, stmt nodes.Node, source string) (*Query, error)
 		}
 	case nodes.UpdateStmt:
 	default:
-		return nil, nil
+		return nil, fmt.Errorf("parseQuery: unsupported statement type: %T", n)
```

Comment:
Maybe add a note that here and above you should file an issue if you hit this error because it's easy to see someone generating code that (should be) valid but hitting an edge case here because we haven't thought about it.

Reply:
That makes sense because with https://github.com/kyleconroy/sqlc/pull/179, these edge cases shouldn't be hit at all. Now the user gets the following error if it happens though:

```
parseQuery: unsupported statement type: pg_query.CreateStmt - feel free to file an issue on GitHub
```